apps/preview/nuxt/pages/showcases/Gallery/GalleryWithBullets.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div class="relative max-h-[600px] flex flex-col h-full gap-1">
+    <SfScrollable
+      class="w-full h-full snap-x snap-mandatory [&::-webkit-scrollbar]:hidden [-ms-overflow-style:none] [scrollbar-width:none]"
+      :active-index="activeIndex"
+      wrapper-class="h-full group/scrollable"
+      is-active-index-centered
+      :previous-disabled="activeIndex === 0"
+      :next-disabled="activeIndex === images.length - 1"
+      buttons-placement="block"
+      @on-prev="
+        () => {
+          activeIndex -= 1;
+        }
+      "
+      @on-next="
+        () => {
+          activeIndex += 1;
+        }
+      "
```

Comment:
Does it work with just this?
```suggestion
      @on-next="activeIndex += 1"
```